### DISCUSSION TITLE
I just wanted to thank you

### DISCUSSION THREAD

**Discussion by tanaybhomia** [rafaelmardojai/blanket#143] [Show and tell]
You dont know, This is my best discover of the year and I love how simple the app is and follows gnome styling also now I dont have to go and search for ambient sounds on internet. why dont you link the social media accounts of the contributers so I can thank them too.

**Comment by rafaelmardojai** [comment]
Thank you for the kind words!

All contributors can be found [here](https://github.com/rafaelmardojai/blanket/graphs/contributors), but most are translators, contributors to the app itself are [Porrumentzio](https://github.com/Porrumentzio) who helped me curating and editing the sounds and [jimmac](https://github.com/jimmac) the icons artist.
